pkg/user/tx_client.go

```diff
@@ -428,7 +531,9 @@ func (client *TxClient) pruneTxTracker() {
 // hash. It will continually loop until the context is cancelled, the tx is found or an error
 // is encountered.
 func (client *TxClient) ConfirmTx(ctx context.Context, txHash string) (*TxResponse, error) {
-	txClient := tx.NewTxClient(client.grpc)
+	// TODO (#4578): Implement logic to potentially use other connections if the primary fails.
```

Comment:
This PR "closes #4578" so it would be nice to open a new issue to track this TODO and any other follow-ups that were created by this PR. Also helpful to track any remaining action items from #4578

Reply:
I'll update #4651

Reply:
I actually don't see why this shouldn't also be included in the scope of the `broadcastMulti` call. I would have just parallelised the top-level call (in `SubmitPayForBlob` and waited for the first one to return `core.TxStatusCommitted`. 

What could be a potential problem with this idea? cc @rootulp

Reply:
No objections especially if it makes it easier to understand the concurrent code